src/Serializer.php

```diff
@@ -321,9 +353,15 @@ protected function getRequestedFields()
         return $fields;
     }
 
+    /**
+     * @return array|null
+     */
     protected function getIncluded()
     {
         $include = $this->request->get($this->expandParam);
+        if (!$this->request->isGet) {
+            return null;
+        }
```

Comment:
@tuyakhov This looks great! I just tested the latest master on a client app and found that this particular change broke things. The app in question assumes that a `POST /articles?include=author` would properly return the author resource in addition to the full article resource.

From what I've seen discussed in the Ember world with Ember Data (and from what I can see from the following links), that specific approach is indeed allowed ...
- http://discuss.jsonapi.org/t/including-resources-during-creation/1073/2
- http://discuss.jsonapi.org/t/can-we-include-documents-in-a-post/867
- https://github.com/cerebris/jsonapi-resources/blob/master/test/integration/requests/request_test.rb#L884

Thoughts?